kylin-it/src/test/java/org/apache/kylin/job/BaseTestDistributedScheduler.java

```diff
@@ -69,7 +69,6 @@
     @BeforeClass
     public static void setup() throws Exception {
         staticCreateTestMetadata();
-        System.setProperty("kylin.job.lock", "org.apache.kylin.storage.hbase.util.ZookeeperDistributedJobLock");
```

Comment:
In fact, class "ZookeeperDistributedJobLock" already has been renamed to "ZookeeperDistributedLock"